<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInterface</class>
 <widget class="QWidget" name="UserInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="wid_left" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>61</width>
     <height>501</height>
    </rect>
   </property>
   <widget class="MyQLabel" name="lb_usr">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>53</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_chat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>chat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_friend">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>friend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_addFriend">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>add</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="wid_middle" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>191</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="wid_right" native="true">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>0</y>
     <width>391</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyQLabel</class>
   <extends>QLabel</extends>
   <header location="global">myqlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
